src/sources/Dataset.js

```diff
@@ -36,9 +36,8 @@ export default class Dataset extends BaseWindshaft {
      *
      * @fires CartoError
      *
-     * @constructor Dataset
-     * @extends carto.source.Base
```

Comment:
the `@extends` indicated that this could be used in the place of `carto.source.Base`